hello-arc/releases/app/hello-arc.yaml

```diff
@@ -18,4 +18,4 @@ spec:
     replicaCount: 3
     env:
       name: MESSAGE
-      value: This is an Azure Arc GitOps Demo!
+      value: Welcome to ArcBox DevOps GitOps Demo!
```

Comment:
```suggestion
      value: Welcome to Azure Arc GitOps Demo!
```

Reply:
lets leave it generic with Azure Arc GitOps demo since users may deploy this outside of context of ArcBox